Makefile

```diff
@@ -73,7 +73,7 @@ deploy-metallb-secret:
 
 # --- Update CRDs -------------------------------------------------------------
 
-TRAEFIK_HELM_VERSION = v9.12.3
+TRAEFIK_HELM_VERSION = v9.17.6
```

Comment:
This version was discovered by looking at the releases of https://github.com/traefik/traefik-helm-chart and finding the one that updated the chart to the version of traefik this PR uses. It is tedious and manual.